<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMatchDialog</class>
 <widget class="QDialog" name="AddMatchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>711</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Add New Match, pozhaluista</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Stadium</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="stadiumEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Team 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="team1Edit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Team 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="team2Edit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Team 1 Score</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="team1ScoreEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Team 2 Score</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="team2ScoreEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="submitButton">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
